README.md

```diff
@@ -116,29 +108,31 @@ Then go to `http://localhost:8080` in your web browser.
 
 Having problems?  Check out our [Troubleshooting guide](TROUBLESHOOTING.md).
 
-## CUTR Release Process
+## MobilityData Release Process
 
 **Snapshots**
 
-We've set up a Maven repository to hold the snapshot artifacts from this project in a Github project - [cutr-mvn-repo](https://github.com/CUTR-at-USF/cutr-mvn-repo).
-
-At CUTR, we should run the following at the command-line to create a new artifact:
-~~~
-mvn -Dgpg.skip -DaltDeploymentRepository=cutr-snapshots::default::file:"/Git Projects/cutr-mvn-repo/snapshots" clean deploy
-~~~
-
-Then commit using Git and push new artifacts to Github.
+We've set up a Maven repository on GitHub Packages to hold the snapshot artifacts from this project - [GTFS Realtime Validator Packages](https://github.com/orgs/MobilityData/packages?repo_name=gtfs-realtime-validator). The GitHub Action [`test_package.yml`](../.github/workflows/test_package.yml) publishes to this repository.
 
```

Comment:
I would add here:

>*NOTE: You need [to authenticate with GitHub](https://docs.github.com/en/packages/working-with-a-github-packages-registry/working-with-the-apache-maven-registry#authenticating-with-a-personal-access-token) to download the below artifacts*

Reply:
True, thanks for pointing this out! Done in [27e8037](https://github.com/MobilityData/gtfs-realtime-validator/pull/85/commits/27e80373b9cf3e17b208c5d89a8494211ad236ce)